codebases/index.md

```diff
@@ -0,0 +1,37 @@
+
+# Where we work: our codebases
+
+The Foundation for Public Code supports ambitious communities and their codebases to scale effectively, including smoother collaboration and shared governance.
+
+Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
```

Comment:
I suggest reordering them in the order:

- Mature
- Incubation
- Assessment
- Attic

My reasoning is that we want to put the spotlight on those who are mature, and not de-incentivise those who are in early phases to move down the list.

Reply:
Interesting! I suggest we implement this only once we have mature codebases, instead of putting an empty category at the top right now.